ghost/core/test/e2e-webhooks/pages.test.js

```diff
@@ -120,6 +120,59 @@ describe('page.* events', function () {
             });
     });
 
+    it('page.edited event is triggered', async function () {
+        const webhookURL = 'https://test-webhook-receiver.com/page-edited/';
+        await webhookMockReceiver.mock(webhookURL);
+        await fixtureManager.insertWebhook({
+            event: 'page.edited',
+            url: webhookURL
+        });
+
+        const res = await adminAPIAgent
+            .post('pages/')
+            .body({
+                pages: [
+                    {
+                        title: 'testing page.edited webhook',
+                        status: 'draft',
+                        slug: 'testing-page-edited-webhook'
+                    }
+                ]
+            })
+            .expectStatus(201);
+        
+        const id = res.body.pages[0].id;
+        const updatedPage = res.body.pages[0];
+        updatedPage.title = 'updated test page';
```

Comment:
Not something to change here necessarily. Just a small piece of feedback to improve future contributions. It's generally a more readable and maintainable solution when tests can achieve the needed state in as few steps as possible (unless it's some hacky shortcut that is vague to understand). In this case, just editing the title would have triggered the 'page.edited' event. Less code - better code 😊